packages/cactus-plugin-keychain-google-sm/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@hyperledger/cactus-plugin-keychain-google-sm",
+  "version": "0.6.0",
+  "description": "A keychain implementation storing its entries in Azure key vault.",
+  "main": "dist/lib/main/typescript/index.js",
+  "mainMinified": "dist/cactus-plugin-keychain-google-sm.node.umd.min.js",
+  "browser": "dist/cactus-plugin-keychain-google-sm.web.umd.js",
+  "browserMinified": "dist/cactus-plugin-keychain-google-sm.web.umd.min.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/types/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/",
+    "codegen:openapi": "npm run generate-sdk",
+    "codegen": "run-p 'codegen:*'",
+    "watch": "npm-watch",
+    "pretsc": "npm run generate-sdk",
```

Comment:
@jagpreetsinghsasan The `pretsc` script can be safely deleted now with the new build system in place.